platforms/Linux/shared/RPM/createrepo/createrepo_rpm.sh

```diff
@@ -0,0 +1,40 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2021 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+#!/usr/bin/env bash
+
+set -ex
+
+INDIR=/input
+OUTDIR=/output
+if [[ ! -d "$INDIR" ]]; then
+    echo "$INDIR does not exist, unable to copy rpm file!"
+    exit 1
+fi
+if [[ ! -d "$OUTDIR" ]]; then
+    echo "$OUTDIR does not exist, so no place to copy the artifacts!"
+    exit 1
+fi
+
+# always make sure we're up to date
+yum update -y
+
+# prepare direcoties
+mkdir -p $HOME/createrepo/
```

Comment:
this is missing some quoting. Will fail if `$HOME` has spaces etc.

Reply:
Thanks for the feedback! fixed in 5971db0486c3360b6e2a4e92a3cfd0e1ca262c05